service/src/main/java/edu/softserve/zoo/service/config/ServiceTestConfig.java

```diff
@@ -0,0 +1,16 @@
+package edu.softserve.zoo.service.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.context.annotation.Profile;
+
+/**
+ * This Configuration is loaded when the 'test' profile is active
+ *
+ * @author Andrii Abramov on 6/13/16.
+ */
+@Configuration
+@Profile("test")
```

Comment:
using profile on class basis is too poor approach to provide flexibility. use profile on beans instead